cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Poll.hs

```diff
@@ -32,23 +32,22 @@ import qualified System.IO as IO
 import           System.IO (stderr, stdin, stdout)
 
 
-runGovernancePollCmds :: GovernancePollCmds era -> ExceptT GovernanceCmdError IO ()
+runGovernancePollCmds :: GovernancePollCmds -> ExceptT GovernanceCmdError IO ()
```

Comment:
We should retain the `era` type argument.

Reply:
Done :+1: